dbms/src/TestUtils/executorSerializer.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <Common/FmtUtils.h>
+#include <Flash/Statistics/traverseExecutors.h>
+#include <Interpreters/Context.h>
+#include <TestUtils/TiFlashTestException.h>
+namespace DB
+{
+class Context;
+
+namespace tests
+{
+struct ExecutorSerializerContext
+{
+    Context & context;
```

Comment:
why not `FmtBuffer buf;`?
Is it necessary to pass buf in externally?

Reply:
I have removed the ExecutorSerializerContext